### PR TITLE
FIX :[bug #1309]

### PR DESCRIPTION
**Pull request by KreizIT**

### CODE REVIEW

File: htdocs/comm/action/fiche.php [hunk 1]
```diff
@@ -538,7 +538,7 @@ function setdatefields()
 
 	// Busy
 	print '<tr><td width="30%" class="nowrap">'.$langs->trans("Busy").'</td><td>';
-	print '<input id="transparency" type="checkbox" name="transparency" value="'.$actioncomm->transparency.'">';
+	print '<input id="transparency" type="checkbox" name="transparency" >';
```
Comment (rdoursenaud) [inline htdocs/comm/action/fiche.php]: Looks like you're hidding the problem under the carpet.
Shouldn't it be something like, in pseudo-code :

``` php
print '<input id="transparency" type="checkbox" name="transparency"'
if ($actioncomm->transparency) {
 print ' checked';
}
print '>';
```

?

**Comment by KreizIT** [comment]
As it's a "create" action, there's no value for $actioncomm->transparency. 
Input definition for edit is well defined on line 791.

**Comment by rdoursenaud** [comment]
Oh, you're right. I didn't look at the context. My bad.

One feature that would be usefull though is keeping the context in case there is an error when validating the form. I tend to always forget to put a date in there…

**Comment by rdoursenaud** [comment]
Just tested and the code I suggested fixes the bug while keeping the intended feature (prevent form reset on error).

**Comment by KreizIT** [comment]
You're right :)

**Pull request merged**